packages/sdk/src/modules/base/index.ts

```diff
@@ -213,30 +223,31 @@ export class Base {
                       unsubscribeNewHeads();
                     }
 
-                    const eventsTriggeredByTx: any = []
```

Comment:
Previous event logic was returning an empty list. It now returns a list of peaqEvents. The code iterates through the events in the current block to build the peaqEvents object. It checks if any event contains an extrinsic failed. If it does, a transactionError is built out by calling the _transactionError() function.